backend/question-service/lambda/app/scraper/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
can we standardize all env files as `.env.development`

Reply:
Resolved in f38c245482f8f1e717f5d1f41dc82882b7ce0908